src/launch/lib/service/common.py

```diff
@@ -70,3 +70,26 @@ def determine_existing_uuid(
         input_data = recursive_dictionary_merge(input_data, uuid_dict["platform"])
 
     return input_data
+
+
+def load_launchconfig(path=LAUNCHCONFIG_PATH_LOCAL) -> dict:
+    """
+    Load the launchconfig file from the given path.
+    
+    Args:
+        path (str): Path to the launchconfig file
+        
+    Returns:
+        dict: The launchconfig file contents
+        
+    Raises:
+        Exception: If the launchconfig file cannot be read
+    """
+    with open(path, "r") as f:
+        click.secho(
+            f"Reading config file found at local path: {LAUNCHCONFIG_PATH_LOCAL=}"
+        )
+        input_data = json.load(f)
+        input_data = input_data_validation(input_data)
+        return input_data
+    raise Exception("Failed to read launchconfig file")
```

Comment:
The final `raise` statement here doesn't seem like it's in a reachable code path. The `open()` context manager will raise a FileNotFound exception, for example, when provided a filename that doesn't exist. What was the intent here?